src/app/haku/[oid]/valintaryhma/components/valintaryhma-list.tsx

```diff
@@ -69,7 +70,12 @@ const Content = ({
       ? NAV_LIST_SELECTED_ITEM_CLASS
       : '';
 
-  return valintaryhma.alaValintaryhmat.length > 0 ? (
+  const alaValintaryhmatUserHasPermission =
+    valintaryhma.alaValintaryhmat.filter(
+      hasRightToValintaryhmaOrAlaValintaryhma,
```

Comment:
Tämänkin voisi yhdenmukaistaa käyttämään `Permission`-termiä.
```suggestion
      hasPermissionToValintaryhmaOrAlaValintaryhma,
```